### PR TITLE
set origami-type to config

### PR DESCRIPTION
**Pull request by chee**
https://github.com/Financial-Times/origami-navigation-data

### CODE REVIEW

**Review by notlee — APPROVED** [review]
this one doesn't neatly fit, config seems ok 👍

**Comment by notlee** [comment]
Could be library since the makefile contains logic?

**Comment by chee** [comment]
> Could be library since the makefile contains logic?

yes, i think you're right

File: origami.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"description": "Common tools for building and running Origami services",
-	"origamiType": null,
+	"origamiType": "config",
```
Comment (chee) [inline origami.json]:
```suggestion
	"origamiType": "library",
```

**Pull request merged**